apps/server/src/connect/connectApi.test.ts

```diff
@@ -87,4 +89,60 @@ describe("connectApi", () => {
       expect(institution).toEqual(expectedInstitutionResponse);
     });
   });
+
+  describe("loadPopularInstitutions", () => {
+    it("expects getRecommendedInstitutions to be called with filterTestBanks = true if ENV is prod", async () => {
+      jest.spyOn(config, "getConfig").mockReturnValueOnce({
+        ENV: "prod",
+      });
+
+      jest.spyOn(preferences, "getPreferences").mockResolvedValueOnce({
```

Comment:
These tests are really mocky. Still parsing through the rest of the changes to determine a suggestion.

Reply:
Looks like if you follow my other suggestion then we can get rid of the mockiness in these tests